### PR TITLE
[check] Allow skipping line check for code samples and return all errors #70 #74

### LINKED ISSUE #70: [check] Allow skipping line checks for code samples
In code samples we should allow a higher length of lines, I think `114` is good.

### LINKED ISSUE #74: [qa] Line length check should return all errors
The line length check should return all errors, now it's stopping at the first found.

### PR DESCRIPTION
**Pull request by ppabcd**
Check code samples. If it is code samples set line height to 114.

Fixes #70 
Fixes #74

### CODE REVIEW

File: check_line_length.py [hunk 1]
```diff
@@ -24,14 +26,30 @@ def check_line_length(file_path):
     interrupts execution with exit code 1 otherwise
     """
     file = file_path.split('/')[-1]
+    limit_type = 'text'
     with open(file_path) as f:
         lines = f.readlines()
     for (line_number, line) in enumerate(lines, start=1):
+        is_code = re.search(r'code-block', line)
```
Comment (atb00ker) [inline check_line_length.py]: I am sure if i understand this correctly.
You are trying to check if 'code-block' is in the line?
But, the code-block might not have 'code-block' written. ( It most probably never have that. )

Code-block is:
```
Hello i am a code-block!
```

Comment (ppabcd) [reply check_line_length.py]: ok tomorrow i'll create for 
```
this
```

Comment (ppabcd) [reply check_line_length.py]: i see about code block from here @atb00ker
http://www.sphinx-doc.org/en/1.5/markup/code.html

Comment (ppabcd) [reply check_line_length.py]: and this is not using markdown but using reStructuredText. I only see using `.. code-block::programming` for checked code example.

File: check_line_length.py [hunk 2]
```diff
@@ -7,6 +7,8 @@
 import sys
 
 LIMIT = 75
+LIMIT_TEXT = 75
```
[the remark this answers is not in the document]
Comment (ppabcd) [reply check_line_length.py]: hmm ok maybe only using `LIMIT_TEXT` and `LIMIT_CODE` for getting line height. I'll change.

**Comment by ppabcd** [comment]
Thank you @nemesisdesign

**Pull request merged**